### ISSUE TITLE
Movie certifications not returned with allocine

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#2669]
Original [issue 2672](https://code.google.com/p/moviejukebox/issues/detail?id=2672) created by Omertron on 2013-08-31T09:48:44.000Z:

<b>What steps will reproduce the problem?</b>
1.with mjb.internet.plugin=com.moviejukebox.plugin.AllocinePlugin in moviejukebox.properties you will get a certificaton of &lt;certification source=&quot;ALLOCINE&quot;&gt;All&lt;/certification&gt; in the jukebox/whatever_movie.xml file. This is true for ALL movies. 
This problem appeared when allocine switched to a secured API about one year ago maybe. Movies details were then OK except certification that never worked, whereas they were working very well before the allocine change. So I don't know if allocine.webservice doesn't provide certifications anymore or if it's the yamj.API.lib that just does not handle it correctly
2. The consequence of the above is that the parental control function of my skin is not working at all (see note at the bottom, but this is NOT the scope of this defect report). This is just to explain that my usage of certifications is for parental control, that's why it's important for me. 
<b>3.</b>

<b>What is the expected output? What do you see instead?</b>
&quot;All&quot; should be replaced by 06,12,14, 16,18, TV-MA, U ... depending on the movie certification. It is working this - right - way with IMDB for movies or TVDB for TV Series. The problem is only with allocine for movies.
expected outpout is &lt;certification source=&quot;ALLOCINE&quot;&gt;12&lt;/certification&gt; for example
and I see &lt;certification source=&quot;ALLOCINE&quot;&gt;All&lt;/certification&gt; instead

<b>What version/revision of YAMJ are you using? On what operating system?</b>
latest one as of today: Version: 2.10-SNAPSHOT Revision: r3731 . This version includes allocine-api-1.3.jar lib
It is used on imac OSX 10.6.8 via Tomy 1.4.1

<b>What skin are you using?</b>
Aeon beffa big poster, but I've made checks and I've got the exact same result with aeon_mod_jlb, Aio_SabishGT, default and gfb107
so it's not a problem of the skin but of interpretation/output from the allocine-api lib writing in jukebox/movies.xml files

<b>What device are you using YAMJ on? Networked Media Tank, PlayonHD, Other?</b>
Popcorn C200 with Diskstation' NAS

<b>Please provide any additional information below.</b>
Full description of the allocine certification defect and my tests on http://www.networkedmediatank.com/showthread.php?tid=65049
I want to thank jluc2808 who did put me on the right track via private messages. He helped me understand that I should check the xml details in the jukebox instead of complaining about the problem of parental control not working. That was in fact just a result of not having the certification
As a result when the allocine certification will be fixed I should also be able to close the parental control thread I created on http://www.networkedmediatank.com/showthread.php?tid=66954

In summary: Please adjust the allocine interpretation by allocine-api-1.3.jar lib in order to handle certifications

thank-you
